### DISCUSSION TITLE
About Language Packs

### DISCUSSION THREAD

**Discussion by mowumo** [audacity/audacity#4574] [Development]
### Bug description

Thank you for bringing such a great work to me. During the process of using it, I found that there were still incomplete translations in the software. If they could be improved in the next version, it would be even more helpful for Chinese speakers to use it. Thank you.

### Steps to reproduce

..

### Expected behavior

..

### Audacity Version

current master / alpha / nightly build

### Operating system

Windows

**Comment by LWinterberg** [comment]
We rely on community translators for our translations. Some translations are incomplete, you can improve them as described here: https://support.audacityteam.org/community/contributing/translating
